src/App.js

```diff
@@ -6,23 +6,43 @@ import './App.css'
 
 class App extends Component {
   state = {
-    accountId: 2,
-    icon: '-T' //test
+    accountId: '',
+    icon: ''
+  }
+
+  setAccountInfo = state => {
+    this.setState(state)
+  }
+
+  handleSignOut = () => {
+    this.setState({ accountId: '' })
+    window.history.back()
```

Comment:
```suggestion
    window.history.push('/login')
```

We should push them directly to the login page

Reply:
sure, made the change